src/coreclr/.nuget/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.pkgproj

```diff
@@ -0,0 +1,27 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
```

Comment:
Why are you skipping the package validation?

Reply:
I don't know what this flag does. I just copied that from the [Microsoft.ILVerification.pkgproj](https://github.com/dotnet/runtime/blob/main/src/coreclr/.nuget/Microsoft.ILVerification/Microsoft.ILVerification.pkgproj), I will try to eliminate this flag and see if it still builds.

Reply:
It worked, both in local and official build, so I will simply remove that property to avoid skipping validation.